build.gradle

```diff
@@ -17,7 +19,7 @@ apply plugin: "jacoco"
 
 buildscript {
     ext {
-        appengineVersion = "1.9.54"
+        appengineVersion = "1.9.56"
         jsoupVersion = "1.10.2"
         checkstyleVersion = "6.19"
```

Comment:
Since the `dependencies` block no longer contains a reference to `appengineVersion`, all the version numbers can be bumped back to the root level and the `ext` block removed, like how it was prior to 2a7638e46de40e46d7806745bd7b34b876248c42.